pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
https://gist.github.com/zsyed91/77dce9db6c46d2237532

Let me know if you need anymore clarifications. I'd be glad to explain it further. Here is a working example to get an idea on repl.it:

https://repl.it/Bxdv/2